.github/workflows/concrete_python_tests_linux.yml

```diff
@@ -81,6 +81,8 @@ jobs:
             set -e
             rm -rf /build/*
 
+            dnf -y install graphviz graphviz-devel
```

Comment:
Should be added to the docker image no ?

Reply:
Would just changing the Dockerfiles work?

Reply:
Does not seem to work.

Reply:
No the docker image has to be generated also. So it's a two PR.